src/pyghidra_mcp/context.py

```diff
@@ -186,6 +229,19 @@ def get_program_info(self, binary_name: str) -> "ProgramInfo":
             raise ValueError(
                 f"Binary {binary_name} not found. Available binaries: {available_progs}"
             )
+        if not program_info.analysis_complete:
```

Comment:
This check allows this condition to be handled well. Before it would show empty lists. This will protect all tool calls that are specific to a bin. 

<img width="859" height="583" alt="image" src="https://github.com/user-attachments/assets/7c489614-9212-4278-9ea5-cf230a7fbc33" />